### PR TITLE
update account en

### PR DESCRIPTION
**Pull request by bxq2011hust**

### CODE REVIEW

File: en/docs/installation.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ Type 'help' or 'h' for help. Type 'quit' or 'q' to quit console.
 =============================================================================================
 ```
 
-### Acquire information through console
+### Query blockchain's information
```
Comment (cyjseagull) [inline en/docs/installation.md]: Recommend to add `console`, eg. `Query blockchain's information through console`

File: en/index.rst [hunk 2]
```diff
@@ -53,7 +53,6 @@ FISCO BCOS is a reliable, secure, efficient and portable blockchain platform wit
    docs/enterprise_tools/index.md
    docs/sdk/sdk.md
    docs/browser/browser.md
-   docs/design/index.rst
```
Comment (cyjseagull) [inline en/index.rst]: Why delete this?

Comment (bxq2011hust) [reply en/index.rst]: > Why delete this?

We don't check this, so hide it @qyan-dev

File: en/docs/installation.md (hunk 1 above)
Comment (bxq2011hust) [reply en/docs/installation.md]: the whole chapter named `Using console`,  so I think that there is no need to add `through the console `

File: en/docs/tutorial/key_concepts.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Core concept
+# Core concept (Work in progress)
```
Comment (qyan-dev) [inline en/docs/tutorial/key_concepts.md]: Ditto

File: en/docs/enterprise_tools/index.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# Deployment tool
+# Enterprise deployment tool
```
Comment (qyan-dev) [inline en/docs/enterprise_tools/index.md]: Experience enterprise deployment toolkit

File: en/docs/tutorial/account.md [hunk 5]
```diff
@@ -1,19 +1,16 @@
 # Create and manage accounts
```
[the remark this answers is not in the document]
Comment (qyan-dev) [reply en/docs/tutorial/account.md]: Manage blockchain accounts

**Pull request merged**